src/app/gallery/[type]/page.js

```diff
@@ -0,0 +1,27 @@
+import Notfound from "@/app/not-found";
+import Title from "@/components/Title";
+
+const Page = ({ params }) => {
+  const PAGES = {
+    fall: "Fall",
+    winter: "Winter",
+    spring: "Spring",
+    past: "Past",
+  };
+  if (!PAGES[params.type]) {
+    return (
+      <div className="text-7xl">
+        {" "}
+        <Notfound />{" "}
+      </div>
```

Comment:
you can import a notFound() function from next library. (return the function directly, with no dividers).

https://nextjs.org/docs/app/api-reference/functions/not-found